### PR TITLE
Add support for greyscale SSD1362 OLED (256x64)

### PR DESCRIPTION
**Pull request by rm-hull**
Supercedes #267

## TODO
- [x] Driver code
- [x] Add documentation
- [x] Add PDF datasheet
- [x] Regression tests
- [ ] Add `ssd1362.conf` to luma.examples
- [x] Add entry to changelog
- [x] Ready for review

## Notes
Breakout board and SSD1362 kindly supplied by @xcarcelle

`ssd1362.conf` settings:

```
--display=ssd1362
--interface=spi
--width=256
--height=64
--spi-bus-speed=16000000
```

### Breakout board pinouts

image

image

| OLED Pin | OLED Name | RPi Pin | RPi Function | Remarks |
|----------|----------|---------|----------|-------|
| 1 | CS | P01-24 | GPIO 8 (CE0) | Chip Select |
| 2 | RST | P01-22 | GPIO 25 | Reset |
| 3 | DC | P01-18 | GPIO 24 | Data/Command |
| 4 | SCLK | P01-23 | GPIO 11 (SCLK) | Serial Clock |
| 5 | SDA | P01-19 | GPIO 10 (MOSI) | Serial Data |
| 6 | GND | P01-20 | GND | Ground |
| 7 | VCC | P01-17 | 3V3 | +3.3V Power |

### Reference
* https://www.andersdx.com/media/downloads/displays/monochrome-displays/oled-displays/OEL9M1020-O-E/OEL9M1020-O-E.pdf

### CODE REVIEW

**Comment by rm-hull** [comment]
Looks like there is some offsetting, mirroring and nibble alignment needed, but it is looking promising:
image

Starting to get recognisable shapes now .. this is what it should look like (but in greyscale)
luma_000002

**Comment by rm-hull** [comment]
image

File: README.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ luma.oled
 
 Luma.OLED
 ---------
-**Display drivers for SSD1306 / SSD1309 / SSD1322 / SSD1325 / SSD1327 / SSD1331 / SSD1351 / SH1106**
+**Display drivers for SSD1306 / SSD1309 / SSD1322 / SSD1325 / SSD1327 / SSD1331 / SSD1351 / SSH1362 / SH1106**
```
Comment (thijstriemstra) [inline README.rst]: should be ssd1362

File: tests/test_ssd1362.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014-18 Richard Hull and contributors
```
Comment (thijstriemstra) [inline tests/test_ssd1362.py]: should be 2020

**Comment by rm-hull** [comment]
3.4.0 released to Pypi with this change included: https://pypi.org/project/luma.oled/3.4.0/

**Comment by nikitalita** [comment]
@thijstriemstra @xcarcelle  Where did you get that breakout board? I can't find that for sale anywhere.

**Pull request merged**